nx-X11/programs/Xserver/hw/nxagent/Keystroke.c

```diff
@@ -418,19 +424,20 @@ static enum nxagentSpecialKeystroke find_keystroke(XKeyEvent *X)
   enum nxagentSpecialKeystroke ret = KEYSTROKE_NOTHING;
 
   while (cur->stroke != KEYSTROKE_END_MARKER) {
-    if (cur->keysym == keysym && modifier_matches(cur->modifierMask, cur->modifierAltMeta, X->state)) {
+    if (cur->keysym == *keysym && modifier_matches(cur->modifierMask, cur->modifierAltMeta, X->state)) {
```

Comment:
I _think_ as XGetKeyboardMapping returns a mapping we should use an index here to reference the data. Even if that index is always 0. Same at the other places.

Citing the mangpage:

```
The number of elements in the KeySyms list is:

       keycode_count * keysyms_per_keycode_return

       KeySym number N, counting from zero, for KeyCode K has the following index in the list, counting from zero:
       (K - first_code) * keysyms_per_code_return + N
```